crash on launching

fresh build


```
2021-02-01|12:52:23.510 [INFO ] Syncing to state (32850,32850,32850,32851,1) from peer 144.202.82.184:19130 cur graph state:(0,0,0,1,1) module=blkmanager
panic: DAG can't find intersection!

goroutine 18 [running]:
github.com/btceasypay/bitcoinpay/core/blockdag.(*Phantom).updateMainChain(0xc0000f3a40, 0xc00049e9c0, 0xc00049e9c0, 0xc00049e9c0)
        /home/tony/github.com/btceasypay/bitcoinpay/core/blockdag/phantom.go:290 +0x44f
github.com/btceasypay/bitcoinpay/core/blockdag.(*Phantom).AddBlock(0xc0000f3a40, 0xeb4960, 0xc00049e9c0, 0x0)
        /home/tony/github.com/btceasypay/bitcoinpay/core/blockdag/phantom.go:66 +0xb3
github.com/btceasypay/bitcoinpay/core/blockdag.(*BlockDAG).AddBlock(0xc00009a580, 0xea98a0, 0xc0004120f0, 0x0, 0x0, 0x0)
        /home/tony/github.com/btceasypay/bitcoinpay/core/blockdag/blockdag.go:289 +0x55b
github.com/btceasypay/bitcoinpay/core/blockchain.(*BlockChain).maybeAcceptBlock(0xc00011aa00, 0xc00044e540, 0x4, 0x0, 0x0)
        /home/tony/github.com/btceasypay/bitcoinpay/core/blockchain/accept.go:124 +0x46d
github.com/btceasypay/bitcoinpay/core/blockchain.(*BlockChain).ProcessBlock(0xc00011aa00, 0xc00044e540, 0xc000000004, 0xc0001c0d68, 0x1, 0x2)
        /home/tony/github.com/btceasypay/bitcoinpay/core/blockchain/process.go:139 +0x4a8
github.com/btceasypay/bitcoinpay/services/blkmgr.(*BlockManager).handleBlockMsg(0xc00011a900, 0xc000021500, 0x17)
        /home/tony/github.com/btceasypay/bitcoinpay/services/blkmgr/handle_block.go:73 +0x27b
github.com/btceasypay/bitcoinpay/services/blkmgr.(*BlockManager).blockHandler(0xc00011a900)
        /home/tony/github.com/btceasypay/bitcoinpay/services/blkmgr/blkmgr.go:519 +0xc99
created by github.com/btceasypay/bitcoinpay/services/blkmgr.(*BlockManager).Start
        /home/tony/github.com/btceasypay/bitcoinpay/services/blkmgr/blkmgr.go:327 +0xbb
```

--- a/core/blockdag/phantom.go
+++ b/core/blockdag/phantom.go
@@ -45,7 +45,18 @@ func (ph *Phantom) Init(bd *BlockDAG) bool {
 
 	ph.bd.order = map[uint]uint{}
 
+	// main chain
 	ph.mainChain = &MainChain{bd, MaxId, 0}
+	ph.bd.db.Update(func(dbTx database.Tx) error {
+		meta := dbTx.Metadata()
+		mchBucket := meta.Bucket(dbnamespace.DagMainChainBucketName)
+		var err error
+		if mchBucket == nil {
+			_, err = meta.CreateBucket(dbnamespace.DagMainChainBucketName)
+		}
+		return err
+	})
+
 	ph.diffAnticone = NewIdSet()
 
 	//vb

